src/AppInitializer.tsx

```diff
@@ -122,8 +122,6 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
   componentDidUpdate(prevProps: Props) {
     // https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/guides/scroll-restoration.md
```

Comment:
Looks like comment link is related to scroll, so it can also probably also be removed.